src/main/java/Duke.java

```diff
@@ -1,10 +1,118 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASKS = 100;
+    public static Task[] list =  new Task[MAX_TASKS];
+    public static int size = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+
+        printWelcomeMessage();
+
+        String userInput = in.nextLine();
+        while (userInput.equals("bye") == false) {
+            if (userInput.equals("list")) {
+                printList();
+            } else if(userInput.contains("done")){   //cannot use command.equals since user is entering more than just "done"
+                isCompleted(userInput);
+            } else {
+                addTask(userInput);
+            }
+            userInput = in.nextLine();
+        }
```

Comment:
I think you did a good job~ You add sperate functions and use understandable function names, it's readable and logical.